code/game/area/area.dm

```diff
@@ -60,6 +60,9 @@
 	///Used to decide what the maximum time between ambience is
 	var/max_ambience_cooldown = 120 SECONDS
 
+	///What sound eviroment to use
+	var/sound_environment = SOUND_ENVIRONMENT_ROOM
```

Comment:
ROOM as the default is bad because it has a reverb
```suggestion
	///What sound environment to use
	var/sound_environment = SOUND_ENVIRONMENT_NONE
```

Reply:
ah ok